docs/_includes/input-chips.html

```diff
@@ -0,0 +1,344 @@
+<div id="input-chips">
+    {% include section-header.html name="input-chips" version=page.versions.input-chips %}
+
+    <p>Input chips are creators and managers of <span class="highlight">chip</span> components. They allow user search of existing chip options, additions, and removals of chips. The search/selection follows the pattern of the combobox pattern.</p>
+    
+    <h3>Basic Input Chips</h3>
+    <p>This is the simplest type of implementation and includes only the bare essentials for using the module.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="input-chips">
+                <div class="input-chips__container">
+                    <div class="input-chips__content">
+                        <div class="input-chips_items" role="group" aria-describedby="input-chips-1-heading">
+                            <span class="chip">
+                                <span id="chip-interactive-1-1-text" class="chip__text">Football</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-1-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-1-2-text" class="chip__text">Basketball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-2-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-1-3-text" class="chip__text">Baseball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-1-3-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                        </div>
+                        <button type="button" class="icon-btn icon-btn--small input-chips_list-toggle">
```

Comment:
- This button needs to have appropriate label to convey to SR users. 
- Also, I noticed chevron not reflecting appropriately when focus in exited using keyboard i.e chevron points up even though the listbox is closed.
- The input is properly conveying that a popup listbox is associated. Do we also need to convey expanded/collapsed state on the chevron button?

Reply:
> This button needs to have appropriate label to convey to SR users.

Good catch! Done.

> Also, I noticed chevron not reflecting appropriately when focus in exited using keyboard i.e chevron points up even though the listbox is closed.

That's a known issue. This is an interaction piece that is more appropriate to handle in a js library, in our case, ebay ui core.

> The input is properly conveying that a popup listbox is associated. Do we also need to convey expanded/collapsed state on the chevron button?

Done. Let me know how that looks. I tried to mimic the same attributes.